web-ui/src/main/resources/catalog/components/elasticsearch/EsFacet.js

```diff
@@ -37,40 +37,44 @@
         includes: [
           "uuid",
           "id",
-          "creat*",
-          "group*",
+          "groupOwner",
           "logo",
-          "category",
-          "cl_topic*",
-          "inspire*",
-          "resource*",
-          "draft*",
-          "overview.*",
-          "owner*",
-          "link*",
-          "image*",
+          "cat",
+          "inspireThemeUri",
+          "inspireTheme_syn",
+          "cl_topic",
+          "resourceType",
+          "resourceTitle*",
+          "resourceAbstract*",
+          "draft",
+          "owner",
+          "link",
           "status*",
           "rating",
-          "tag*",
           "geom",
-          "contact*",
-          "*Org*",
-          "hasBoundingPolygon",
+          "Org*",
           "isTemplate",
           "valid",
           "isHarvested",
           "dateStamp",
-          "documentStandard",
```

Comment:
We use this field in a project to display in the editor board results the metadata standard. As this is hardcoded in javascript we would like to keep it.

Reply:
indeed it can be used quite often also when dispatching views based on it. Same for standardName. I'll change that